src/components/InputPassword.js

```diff
@@ -9,6 +9,8 @@ const zoomFade = keyframes`
   100% { transform: scale(2); opacity: 0; }
 `;
 
+const SInput = styled(Input)`width: 100%;`;
```

Comment:
These keep popping up, can we set `width: 100%` on `input` itself so we don't need to use `SInput`'s ?

Reply:
yeah i was having issues without it but i've just solved it, i can't find any other places where it exist though did i miss any?